### DISCUSSION TITLE
2021-29: 溧阳天目湖一日游

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#227] [Announcements]
# 2021-29: 溧阳天目湖一日游

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2021-29/

**Comment by STRRL** [comment]
好奇雪糕撕开包装以后张什么样子鸭！

**Reply by Xuanwo** [reply]
长这样！

![402521132(1)](https://user-images.githubusercontent.com/5351546/126942574-2f331f7d-edae-4ee6-af66-2ed15d806474.jpg)

**Reply by izackwu** [reply]
看上去就很好吃！不过这名字怎么感觉是在模仿重庆的「南山一棵树」景点啊😂
